src/target/riscv/riscv-011.h

```diff
@@ -0,0 +1,13 @@
+#ifndef OPENOCD_TARGET_RISCV_RISCV_011_H
```

Comment:
Please, add the SPDX license marker to the top of the file.

Also check the other newly created files - I have not commented on all of them.

```suggestion
/* SPDX-License-Identifier: GPL-2.0-or-later */

#ifndef OPENOCD_TARGET_RISCV_RISCV_011_H
```

Reply:
Addressed.